nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/server/AppServerStartup.java

```diff
@@ -543,9 +543,11 @@ public synchronized void stop() {
     }
 
     /**
-     * Proceed to the given run level.
+     * Proceed to the given run level using the given {@link AppServerActivator}.
      *
      * @param runLevel   the run level to proceed to
+     * @param activator  an {@link AppServerActivator activator} used to
+     *                   activate/deactivate the services
      * @return false if an error occurred that required server shutdown; true otherwise
      */
     private boolean proceedTo(int runLevel) {
```

Comment:
Is there a parameter AppServerActivator missing? At least you have added the comment about one.